apps/dotcom/client/e2e/fixtures/ErrorPages.ts

```diff
@@ -0,0 +1,30 @@
+import { Locator, Page } from '@playwright/test'
+import { expect } from '../fixtures/tla-test'
+
+export class ErrorPage {
+	public readonly notFound: Locator
+	public readonly pageNotFound: Locator
+	public readonly privateFile: Locator
+
+	constructor(public readonly page: Page) {
+		this.notFound = this.page.getByRole('heading', { name: 'Not found' })
+		this.pageNotFound = this.page.getByRole('heading', { name: 'Page not found' })
```

Comment:
Think we should unify these two error pages?

Reply:
it's a good call out, you can do it in this PR if you like, but feel free to land without doing so - i can take care of it in my other error page PR